archive/100-theorems-list/70_perfect_numbers.lean

```diff
@@ -0,0 +1,671 @@
+import data.nat.totient
```

Comment:
This needs a copyright header (above the imports) and a module doc (below the imports).

Reply:
I've added those, they'll be in my next commit, although probably with not enough said in the module doc yet.